app/src/main/java/pm/gnosis/heimdall/ui/safe/main/SafeMainActivity.kt

```diff
@@ -284,7 +285,8 @@ class SafeMainActivity : ViewModelActivity<SafeMainContract>() {
                 layout_safe_main_selected_safe_info.text = null
                 layout_safe_main_selected_safe_icon.visible(false)
                 layout_safe_main_selected_safe_progress.visible(false)
-                layout_safe_main_toolbar_title.text = getString(R.string.your_safe)
+                layout_safe_main_toolbar_title.text = getString(R.string.welcome)
+                layout_safe_main_toolbar_icon.visible(false)
```

Comment:
Why are we hiding this? zpl.io/awY6Wzn

Reply:
There's no toolbar icon on that screen.

Reply:
ahh the toolbar icon is the safe icon next to the toolbar title .... kk